jetty-ee11/jetty-ee11-proxy/src/test/java/org/eclipse/jetty/ee11/proxy/AsyncMiddleManServletTest.java

```diff
@@ -1715,7 +1715,7 @@ private void sleep(long delay)
         }
         catch (InterruptedException x)
         {
-            throw new RuntimeIOException(x);
+            throw new UncheckedIOException(new IOException(x));
```

Comment:
```suggestion
            throw new UncheckedIOException(new InterruptedIOException(x));
```

Reply:
IIE doesn't take a cause, so I wont fix here. It is just a test